Classes/EventStore/EventStreamResolver.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Neos\Cqrs\EventStore;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Cqrs\Domain\AggregateRootInterface;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Package\PackageManagerInterface;
+use TYPO3\Flow\Reflection\ClassReflection;
+use TYPO3\Flow\Utility\TypeHandling;
+
+/**
+ * Central authority for resolving event stream names
+ *
+ * @Flow\Scope("singleton")
+ */
+class EventStreamResolver
```

Comment:
Do you think that's a good name for this? Actually it's the EventStream*Name*Resolver but I found that a bit extreem